app/next-client-app/components/data-table/index.tsx

```diff
@@ -39,6 +39,8 @@ interface DataTableProps<TData, TValue> {
   clickableRow?: boolean;
   viewColumns?: boolean;
   paginated?: boolean;
+  overflow?: boolean;
+  RefreshJobsButton?: JSX.Element;
```

Comment:
Yes, we can pass this button as `Filter`, but in this way, we will not have the `filter` component to search the tables any more. 
We can't combine the button and the Filter component, because the `Filter` needs to be on the `page.tsx` which is "server-side" (for it to push the param to URL and retrieve the data), and the `Refresh button` should be on `client-side` (because it need useState).
So should we sacrifice the `Filter` for this?